includes/AssignPost.php

```diff
@@ -98,7 +98,9 @@
 echo "<table>";
 while($row = mysqli_fetch_row($res))
 {
-	echo "<tr><td><h2>$row[0] :</h2></td><td> <input type='text' name=$row[0] value=\"$row[1]\" ></td></tr> <br>";
+	$newRow = preg_replace('/\s+/', '_', $row[0]);
+	$newRow = str_replace('.', '$', $newRow);
+	echo "<tr><td><h2>$row[0] :</h2></td><td> <input type='text' name=\"$newRow\" value=\"$row[1]\" ></td></tr> <br>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->